core/src/tui_core/styles/stylesheet.rs

```diff
@@ -73,3 +73,63 @@ impl Stylesheet {
     }
   }
 }
+
+/// Macro to make building `StyleSheet` easy
+///
+/// Use case:
+/// ```rust
+/// use crossterm::event::*;
+/// use r3bl_rs_utils_core::{CommonResult, Stylesheet};
+///
+/// fn create_stylesheet() -> CommonResult<Stylesheet> {
+///   throws_with_return!({
+///     stylesheet! {
+///         style! {
+///           id: style1
+///           margin: 1
+///           color_bg: Color::Rgb { r: 55, g: 55, b: 248 }
+///         },
+///         vec![
+///             style! {
+///                 id: style1
+///                 margin: 1
+///                 color_bg: Color::Rgb { r: 55, g: 55, b: 248 }
+///             },
+///             style! {
+///                 id: style2
+///                 margin: 1
+///                 color_bg: Color::Rgb { r: 85, g: 85, b: 255 }
+///             },
+///         ]
+///     }
+///   })
+/// }
+/// ```
+/// Note that this operation may panic if the styles could not be added to the Stylesheet
+///
+#[macro_export]
+macro_rules! stylesheet {
+    ($($style:expr),*) => {
+        {
+            trait AddStyle {
+                #[inline]
+                fn add_to_style_sheet(self, stylesheet: &mut Stylesheet);
```

Comment:
Change to `fn add_to_style_sheet(self, stylesheet: &mut Stylesheet) -> CommonResult<()>;` to take advantage of error handling. Panic only when absolutely necessary & it is not necessary to do so here.

Best practices - Generally panics are reserved for total system failures. When at all possible, we want to use error handling to gracefully recover. The program should NOT crash because of some stylesheet issues.

Reply:
Returning `CommonResult<()> ` actually brings a warning about unused result or so, so I didn't want that warning showing that's why I removed it. And going through the source code, `add_styles` always returns `Ok(())` so I felt it was okay to `unwrap` the result regardless